packages/keystone/package.json

```diff
@@ -31,7 +31,7 @@
     "express-pino-logger": "^4.0.0",
     "falsey": "^1.0.0",
     "fs-extra": "^7.0.0",
-    "globby": "^11.0.0",
+    "globby": "^11.0.1",
```

Comment:
The package version bumps here should result in a changeset for all the affected packages. In general it's probably best to leave and version bumps out of a feature PR like this unless they're actually required.